tests/unit/core/config/test_user_config.py

```diff
@@ -15,6 +22,12 @@
 
 @pytest.mark.usefixtures("isolated_fs")
 class TestUserConfig:
```

Comment:
Have you tested the case where the `until` value is not a valid string? Do we get a user-friendly error message?

Reply:
We get the same message from marshmallow as secret/sca:
```
Error: iac: 
  ignored_policies: 
    0: 
      until: ['Not a valid datetime.']
```

However when setting a number as the until date, the error message is messy, so I've added an explicit cast to str for the value to get the same message as above.

Reply:
Looks good enough :+1:.